src/core/openthread-core-default-config.h

```diff
@@ -862,4 +860,68 @@
 #define OPENTHREAD_CONFIG_STAY_AWAKE_BETWEEN_FRAGMENTS          0
 #endif
 
+
+/*
+ * @def OPENTHREAD_CONFIG_ENABLE_DEBUG_UART
+ *
+ * Enable the "Debug Uart" platform feature.
+ *
+ * In the embedded world, the CLI application uses a UART as a console
+ * and the NCP application can be configured to use either a UART or
+ * a SPI type device to transfer data to the host.
+ *
+ * The Debug UART is or requires a second uart on the platform.
+ *
+ * The Debug Uart has two uses:
+ *
+ * Use #1 - for random 'debug printf' type messages a developer may need
+ * Use #2 (selected via DEBUG_LOG_OUTPUT) is a log output.
+ *
+ * See #include <openthread/platform/debug_uart.h> for more details
+ */
+#ifndef OPENTHREAD_CONFIG_ENABLE_DEBUG_UART
+#define OPENTHREAD_CONFIG_ENABLE_DEBUG_UART                     0
+#endif
+
+/**
+ * @def OPENTHREAD_CONFIG_LOG_OUTPUT
+ *
+ * Selects if, and where the LOG output goes to.
+ *
+ * There are several options available, by default there is NO log.
+ * - @sa OPENTHREAD_CONFIG_LOG_OUTPUT_NONE
+ * - @sa OPENTHREAD_CONFIG_LOG_OUTPUT_DEBUG_UART
+ * - and others
+ * This is effectively an ENUM so it can be #if/#else/#endif tested
+ */
+
+/**
+ * @def OPENTHREAD_CONFIG_LOG_OUTPUT Select the DEFAULT log output method.
+ */
+#if !defined(OPENTHREAD_CONFIG_LOG_OUTPUT)
+#define OPENTHREAD_CONFIG_LOG_OUTPUT    OPENTHREAD_CONFIG_LOG_OUTPUT_NONE
```

Comment:
Follow on [comment link](https://github.com/openthread/openthread/pull/2082/files/540fcba416a6e298b8888cf4b4b5c1c043c4f4bc#r134629419), I suggest default value here to be `OPENTHREAD_CONFIG_LOG_OUTPUT_PLATFORM_DEFINED`.